addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/data/TankerkoenigDetailService.java

```diff
@@ -47,6 +49,9 @@ private String getResponseDetailString(String apikey, String locationID) throws
         String urlcomplete = urlbase + "id=" + locationID + "&apikey=" + apikey;
         String response = "";
         try {
+            String userAgent = "OpenHAB, Tankerkoenig-Binding Version ";
```

Comment:
the user agent could start with "openHAB" so it is written the same way as everywhere else.